[2024-12-28T03:28:21.881+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: quarta_dag.tsk3 manual__2024-12-28T03:28:19.247260+00:00 [queued]>
[2024-12-28T03:28:21.920+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: quarta_dag.tsk3 manual__2024-12-28T03:28:19.247260+00:00 [queued]>
[2024-12-28T03:28:21.921+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-12-28T03:28:21.921+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-12-28T03:28:21.921+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-12-28T03:28:21.968+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): tsk3> on 2024-12-28 03:28:19.247260+00:00
[2024-12-28T03:28:21.998+0000] {standard_task_runner.py:55} INFO - Started process 1874 to run task
[2024-12-28T03:28:22.005+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'quarta_dag', 'tsk3', 'manual__2024-12-28T03:28:19.247260+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/quartadag.py', '--cfg-path', '/tmp/tmpcrn_lynk']
[2024-12-28T03:28:22.006+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask tsk3
[2024-12-28T03:28:22.226+0000] {task_command.py:388} INFO - Running <TaskInstance: quarta_dag.tsk3 manual__2024-12-28T03:28:19.247260+00:00 [running]> on host 2f43a4cf56a6
[2024-12-28T03:28:22.464+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=quarta_dag
AIRFLOW_CTX_TASK_ID=tsk3
AIRFLOW_CTX_EXECUTION_DATE=2024-12-28T03:28:19.247260+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-12-28T03:28:19.247260+00:00
[2024-12-28T03:28:22.468+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-12-28T03:28:22.469+0000] {subprocess.py:75} INFO - Running command: ['/bin/bash', '-c', 'sleep 5']
[2024-12-28T03:28:22.496+0000] {subprocess.py:86} INFO - Output:
[2024-12-28T03:28:27.511+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-28T03:28:27.578+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=quarta_dag, task_id=tsk3, execution_date=20241228T032819, start_date=20241228T032821, end_date=20241228T032827
[2024-12-28T03:28:27.650+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-12-28T03:28:27.733+0000] {taskinstance.py:2578} INFO - 1 downstream tasks scheduled from follow-on schedule check
